pxblue/react-native-design-patterns
# Add Password "Forms and Validation" design pattern example

Fixes PXBLUE-2298


#### Changes proposed in this Pull Request:
- Add Password "Forms and Validation" design pattern example


#### Screenshots:
![Simulator Screen Shot - iPhone 12 - 2021-08-17 at 07 43 31](https://user-images.githubusercontent.com/13989985/129719953-f9d01d3d-4aa3-4e89-b7b9-03a3f7c71d04.png)
![Simulator Screen Shot - iPhone 12 - 2021-08-17 at 07 43 43](https://user-images.githubusercontent.com/13989985/129719991-8feeaf39-a11f-4e0c-b635-eeb962d31cfe.png)



#### To Test:
- clone project
- run `yarn && yarn start`
- navigate to Forms and Validation > Password
- observe Password Validation design pattern 

Waiting on tests...
